README.md

```diff
@@ -95,7 +97,7 @@ For details on the `json` and `ldap` backends see the [documentation](https://ow
     bin/ocis-reva frontend
     ```
 
-4.  from inside the oC10 repo run the tests:
+1.  from inside the oC10 repo run the tests:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `5`, was `1`](https://app.codacy.com/gh/owncloud/ocis-reva/pullRequest?prid=4997850)